### PR TITLE
Repositories page

### LINKED ISSUE #428: Repositories page design
[Page of repositories ](https://sul-arclight-demo.stanford.edu/repositories)should be separate from landing page, is a list the best format? 

Other institutions have done cards/3 per row, example: https://findingaids.lib.umich.edu/repositories

### PR DESCRIPTION
**Pull request by marlo-longley**
Closes #428 

For review, please check out the branch and make sure all looks good on desktop + mobile !

Screenshot 2024-04-30 at 14 21 04

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -48,3 +48,6 @@ spec/examples.txt
 config/settings.local.yml
 config/settings/*.local.yml
 config/environments/*.local.yml
+
+
+.DS_Store
```
Comment (corylown) [inline .gitignore:53]: @marlo-longley could you add this to your local global gitignore instead? I think that's our preferred practice for IDE/OS files we want to ignore: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Comment (marlo-longley) [reply .gitignore]: Thanks for the link. Removed. @corylown

File: app/views/landing_page/_cards.html.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div class="row card-container">
   <div class="col-lg-4 my-4 my-lg-0">
     <div class="card h-100">
-      <%= image_tag 'locations.png', alt: '', class: 'card-img-top' %>
+      <%= image_tag 'locations.png', alt: 'Image of library interior', class: 'card-img-top' %>
```
Comment (taylor-steve) [inline app/views/landing_page/_cards.html.erb:4]: Genuinely curious where we stand on alt text for things like this. I intentionally left these blank per:

https://html.spec.whatwg.org/multipage/images.html#a-purely-decorative-image-that-doesn't-add-any-information

Comment (marlo-longley) [reply app/views/landing_page/_cards.html.erb:4]: It just seemed sad to include the attribute with a blank string!

File: app/controllers/arclight/repositories_controller.rb [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Arclight
+  # This is almost an exact copy of the Arclight Controller, but we include it here
+  # in order to override the layout and use our own, 'unwrapped' layout
+  # This allows for proper nesting of multiple rows
+  class RepositoriesController < ApplicationController
+    layout 'unwrapped'
```
Comment (marlo-longley) [inline app/controllers/arclight/repositories_controller.rb:8]: Is there a way to override the layout without importing this whole class?

File: app/views/landing_page/_cards.html.erb (hunk 2 above)
Comment (taylor-steve) [reply app/views/landing_page/_cards.html.erb:4]: I certainly don't mind it either way, I just don't want to keep leaving them blank if that's not something we do around here.

Comment (marlo-longley) [reply app/views/landing_page/_cards.html.erb:4]: I can remove them for now. I see the spec you linked to. We could discuss more generally at our next standup.

Comment (marlo-longley) [reply app/views/landing_page/_cards.html.erb]: @taylor-steve removed.

**Pull request merged**